src/Node/DeprecatedNode.php

```diff
@@ -36,20 +36,24 @@ public function compile(Compiler $compiler): void
         $expr = $this->getNode('expr');
 
         if ($expr instanceof ConstantExpression) {
-            $compiler->write('@trigger_error(')
-                ->subcompile($expr);
+            $compiler
+                ->write('trigger_deprecation(\'twig/twig\', \'\', ')
```

Comment:
this should use an empty package name IMO

Reply:
I've made them configurable now.